SportsPlanner.Droid/App.cs

```diff
@@ -0,0 +1,27 @@
+using SportsPlanner.ViewModels;
+using GalaSoft.MvvmLight.Threading;
+
+namespace SportsPlanner.Droid
+{
+    public static class App
+    {
+        private static ViewModelLocator locator;
+
+        public static ViewModelLocator Locator
+        {
+            get
+            {
+                if (locator == null)
+                {
+                    // Initialize the MVVM Light DispatcherHelper.
+                    // This needs to be called on the UI thread.
+                    DispatcherHelper.Initialize();
```

Comment:
i'm not sure 100% how it works, but it definitely should be used in iOS project too.